packages/forms/src/UIForm/Widget/Widget.component.js

```diff
@@ -8,8 +8,61 @@ import defaultWidgets from '../utils/widgets';
 import { getError } from '../utils/errors';
 import { getValue } from '../utils/properties';
 
+function conditionEvaluationStrategy(value, strategy) {
+	if (!strategy) {
+		return value;
+	}
+	switch (strategy) {
+		case 'length':
+			if (value && Array.isArray(value)) {
+				return value.length;
+			}
+			return 0;
+		default:
+			return false;
+	}
+}
+
+function evaluateCondition(properties, condition) {
+	const value = get(properties, condition.path);
+	const actual = conditionEvaluationStrategy(value, condition.evaluationStrategy);
+	return !condition.negate === includes(condition.values, actual);
+}
+
+/**
+ * Evaluate a list (array) of conditions against a contextual set of values (properties).
+ * Condition specification is done through objects with the following attributes:
+ * <ul>
+ *   <li><em>values</em> (required): list of values compared to the evaluated value</li>
+ *   <li><em>path</em> (required): the path used to extract the value from properties
+ *   to use to compare to values</li>
+ *   <li><em>negate</em> (optional): should the evaluation activate this widget when
```

Comment:
`negate = true` is hard to understand.
Let s use a positive action name like `shouldBe` or find a better one

```
+ * @example {
+ *   values:["A", "B"],
+ *   path: "someProp.someArray",
+ *   shouldBe: true,
+ *   strategy: "length"
+ * }
```